commit 6224de73383583ca4c07751ee6fb76c3bc41dbbd (refs/changes/41/1241/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-01-12T11:45:19-08:00 (2 years, 9 months ago)
    
    joyent/node-triton#164 joyent/node-triton#137 change broke 'triton --version' test case

diff --git a/test/integration/cli-basics.test.js b/test/integration/cli-basics.test.js
index b1a285a..6e91c08 100644
--- a/test/integration/cli-basics.test.js
+++ b/test/integration/cli-basics.test.js
@@ -29,7 +29,9 @@ test('triton (basics)', function (tt) {
         h.triton('--version', function (err, stdout, stderr) {
             if (h.ifErr(t, err, 'triton --version'))
                 return t.end();
-            t.ok(/^triton \d+\.\d+\.\d+/.test(stdout));
+            t.ok(/^Triton CLI \d+\.\d+\.\d+/.test(stdout),
+                'version on first line');
+            t.ok(/^https:/m.test(stdout), 'project link in version output');
             t.end();
         });
     });
